### PR TITLE
Feat/config map mod

### PR DESCRIPTION
**Pull request by elreydetoda**
alright, I think this should work for swapping around the map to be a one to many relationship (which'll keep the config a bit cleaner) (reference in #20)

### CODE REVIEW

File: scraper.py [hunk 1]
```diff
@@ -323,10 +323,9 @@ def get_username_from_url(url):
     # Replace username if found in usernames_map
     usernames_map = config.get("usernames_map")
     if usernames_map:
-        username = usernames_map.get(
-            username, # get by the key that should be replaced
-            username) # default to the key if not found
-
+        for key, value in usernames_map:
+            if username == value:
+                return key
```
Comment (kbondarev) [inline scraper.py]: I haven't tested this - but my first impression is that `value` here is of type List. Hence, the comparison should be done differently:

```suggestion
        for key, value in usernames_map:
            if username in value:
                return key
```

Comment (kbondarev) [reply scraper.py]: @elreydetoda can you take a look at this :point_up: 

I might be wrong on this :man_shrugging:

**Comment by elreydetoda** [comment]
Alright, sorry about that :sweat_smile: . This should be good now :grin:

**Comment by elreydetoda** [comment]
thought I was on my main branch and wiped out the commits from this branch :cry: 
submitting a new PR :sweat_smile:

**Pull request closed without merging**